website_sale_checkout_skip_payment/models/website.py

```diff
@@ -11,7 +11,10 @@ class Website(models.Model):
     website_sale_checkout_skip_message = fields.Text(
         string="Website Sale SKip Message",
         required=True,
-        default="Our team will check your order and send you payment information soon.",
+        translate=True,
+        default=lambda s: _(
```

Comment:
I think this is not correct, as this should have all the languages in the default, not just one...

Reply:
I don't quite get you... this is so we can trigger the translation term generation by Odoo. See here for example: https://github.com/OCA/OCB/blob/60933dcefcfba93a9a52439aa227923bde3afeb6/addons/website_event_track/models/event_track_stage.py#L21

Reply:
I mean that we want to have from the beginning all the translations loaded into this field, not only the user's language one.

Reply:
Odoo is not doing this in any of their modules? Maybe if you return a JSON with languages as keys?

Reply:
I can't find the way. A simple write will have this effect:

```python
>>> website.write({"website_sale_checkout_skip_message": {"en_US": "This is english", "es_ES": "Esto es español"}})
>>> website.website_sale_checkout_skip_message
"{'en_US': 'This is english', 'es_ES': 'Esto es español'}"
```

And with good reason, as it goes through the `convert_to_cache` method for that field class. The same goes for the `default_get` method https://github.com/OCA/OCB/blob/60933dcefcfba93a9a52439aa227923bde3afeb6/odoo/fields.py#L1245

So no black magic here, I'm afraid. And as far as I could tell it looks like Odoo just goes by with that limitation. An example would be the code I linked before...

Reply:
Uhm, this is ugly... @HydrionBurst isn't there any way to initialize a field with all the translated strings?